.github/workflows/codegen.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     uses: googleapis/discovery-artifact-manager/.github/workflows/list-services.yml@master
 
   generate:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
This is ubuntu-latest == ubuntu-20.04 as of now. https://github.com/actions/runner-images

Setting exact version reduces the risk of this job suddenly stop working.